src/usr/local/www/vpn_ipsec_mobile.php

```diff
@@ -477,9 +484,22 @@ function login_banner_change() {
 	array(
 		'none' => gettext('none'),
 		'system' => gettext('system'),
+		'radius' => gettext('RADIUS'),
 	)
 ))->setHelp('Source');
 
+$group = new Form_Group('RADIUS Groups');
+$group->addClass('radius_gr');
+
+$group->add(new Form_Input(
+	'radius_groups',
+	'RADIUS Groups',
+	'text',
+	htmlspecialchars($pconfig['radius_groups'])
+))->setHelp('Comma separated list of RADIUS groups for the EAP-RADIUS authentication.');
```

Comment:
For other areas/purposes like GUI auth we require the groups to be configured locally and matched via RADIUS responses. I'd prefer to see this as a multi-select list of existing local groups rather than a free-form box. Perhaps limit the groups that show up (and get used in the backend) to those with appropriate IPsec privileges as well.

Reply:
Hmmm, so as a user, I will have to create all groups that I would like to use here locally on the pfSense box, so that I can select them in the picker?

> Perhaps limit the groups that show up (and get used in the backend) to those with appropriate IPsec privileges as well.

As a user, how would a give a local group "appropriate IPsec privileges"? Thanks for the clarification!

Reply:
> Hmmm, so as a user, I will have to create all groups that I would like to use here locally on the pfSense box, so that I can select them in the picker?

Yes. That's what you have to do for other places that use RADIUS and LDAP group permissions like for GUI authentication. It's so groups obtained remotely can be mapped to groups and privileges used locally.

> As a user, how would a give a local group "appropriate IPsec privileges"? Thanks for the clarification!

Create or edit a group, search for "IPsec" and you'll see the user privilege there.